src/test/resources/artifacts/ESB/connector/config/Epic.properties

```diff
@@ -0,0 +1,33 @@
+proxyDirectoryRelativePath=/artifacts/ESB/config/proxies/Epic/
+requestDirectoryRelativePath=/artifacts/ESB/config/restRequests/Epic/
+
+base=https://open-ic.epic.com/FHIR/api/FHIR/DSTU2
```

Comment:
Move all the properties that are specific to accounts to separate properties file inside "connector/repository/" folder .
Refer [1].

[1]https://github.com/wso2-extensions/esb-connector-googlespreadsheet/pull/13/files#diff-224e68342d61e85c9386bf00d3afe9dfR1